lib/xcprofiler/execution.rb

```diff
@@ -1,3 +1,5 @@
+require 'digest/md5'
+
 module Xcprofiler
   class Execution
     Struct.new('Position', :path, :line, :column)
```

Comment:
By the way, I think this struct should be named `location`. 😅 
(In log from Xcode names it `location`.)